components/v2/delivery/provider/providers/packetery.js

```diff
@@ -48,6 +48,17 @@ const Packetery = ({ errors, delivery, onSelect }) => {
     return null;
   }
 
+  let packetaOptions = {
+    language: "sk",
+    country: "sk",
+  };
+
+  try {
+    packetaOptions = JSON.parse(get(settings, "packetery_widget_settings"));
```

Comment:
@patrikspilka co to spravi, ak je settings prazdny a ak neexistuje ta hodnota? Nepadne ten JSON.parse?